website/Gemfile.lock

```diff
@@ -39,19 +39,19 @@ GEM
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0.6.0)
     erubis (2.7.0)
-    eventmachine (1.2.5)
+    eventmachine (1.2.7)
     execjs (2.7.0)
-    ffi (1.9.25)
-    haml (5.0.4)
+    ffi (1.11.1)
+    haml (5.1.2)
       temple (>= 0.8.0)
       tilt
     hike (1.2.3)
     hooks (0.4.1)
       uber (~> 0.0.14)
     http_parser.rb (0.6.0)
     i18n (0.7.0)
-    json (2.1.0)
-    kramdown (1.16.2)
+    json (1.8.3.1)
```

Comment:
This seems to have broken the website build. See the netlify deployment job details.

Reply:
That is VERY exciting. Lemme take a look.

Reply:
@mkeeler It looks like just locking it back to 2.1.0 fixed it. That adds some startup time when previewing locally, but not much. 

The root problem here is that this project is maybe using different containers for preview vs. deploy, which should probably not be the case?

Reply:
@alvin-huang Do you have any insights into how the netlify deploy job works and if it works any different from the real website deploy?